content/v2.0/reference/glossary.md

```diff
@@ -1075,20 +1075,41 @@ This protocol is used when speed is desirable and error correction is not necess
 
 An implicit block that encompasses all Flux source text in a universe block.
 
+### unix timestamp
```

Comment:
I think we need to keep these separate. Timestamps can come in multiple formats (RFC3339, RFC3339Nano, ISO8601, etc.). I think it's important to explain unix timestamps specifically.

Reply:
Thoughts:
- If someone looks up "timestamp," they want to see all items relevant to InfluxDB timestamps--different formats, precision, etc. 
- If we add "unix timestamp", do we add entry for "RFC3339 timestamp"?
- Would folks who don't know about unix timestamps, know to look up unix timestamp?

Reply:
I don't think it would be a bad idea to define RFC3339 timestamp in the glossary. When it comes to finding unix timestamp, we use it a few places throughout the docs.